arches_lingo/templates/views/components/etl_modules/migrate-to-lingo.htm

```diff
@@ -12,9 +12,29 @@ <h2>RDM Migrator</h2>
         <p class="pad-btm">
             {% blocktrans %}Click the start button to initiate the process for migrating concepts and schemes from the existing RDM to the Lingo resource models. You will need to have Celery running if your instance has more than 500 concepts.{% endblocktrans %}
         </p>
+        <div class="etl-module-component-block">
+            <h4>
+                <label for="resource-model-select" data-bind="text: $root.translations.selectAThesaurusToMigrate"></label>
+            </h4>
```

Comment:
Ah, I see we're doing this all over. It doesn't seem like a header though. div seems better than an h to me...